source/includes/inkling-guide/_schemas.html.md

```diff
@@ -22,10 +22,10 @@ Here are some examples of constrained types:
 schema MyOutput
   UInt8  {0,1,2,3,4}   label,    # a list of UInt8 values
   String {"a", "bc"}   category, # a list of Strings
-  Int64  { 0:5:100 }   x,        # start:step:stop, step= 5,0..100
-  Int64  { 0:100 }     y,        # start:stop, step= 1, 0..100
-  Int64  { 0..100:25 } z,        # start:stop, numsteps=25, step= 4, 0..100
-  Float32 { 0..2:0.5}    a         # gives (0, 0.5, 1.0, 1.5, 2.0)
+  Int64  { 0:5:100 }   x,        # start:step:stop, {0,5,10,...,95,100}
```

Comment:
Do you think these would look better without the spaces on either side of the brackets? I haven't gone through and changed them all yet to be uniform.